doc/manual/src/language/index.md

```diff
@@ -191,12 +191,12 @@ This is an incomplete overview of language features, by example.
  <tr>
   <td>
 
-   <nixpkgs>
+   `<path>`, `<nixpkgs>`
```

Comment:
```suggestion
   `<path>`
```

Reply:
Either that or the concrete thing many people will still encounter in the wild:

```suggestion
   `<nixpkgs>`
```

Reply:
so, what now? I'm confused by your two suggestions.